Add validation for noise channels parameters (#1357)
PennyLaneAI/pennylane: .github/CHANGELOG.md

<h3>New features since last release</h3>

> * Add validation for noise channel parameters, now it raises an `ValueError` when
>   noise parameters are invalid.

Reviewer: Don't forget to add a link back to your PR here!
```suggestion
* Added validation for noise channel parameters. Invalid noise parameters now
   raise a `ValueError`.
  noise parameters are invalid.
```

Author: Yep, forgot about that 😅
